radix-engine/src/ledger/bootstrap.rs

```diff
@@ -518,6 +587,28 @@ pub fn create_genesis(
         });
     }
 
+    // Genesis helper package
+    {
+        let genesis_helper_code = include_bytes!("../../../assets/genesis_helper.wasm").to_vec();
```

Comment:
Can we add a TODO to add access rules to prevent any other user from calling genesis helper